### PR TITLE
Bug 1882752: openstack BM: Remove MachineSet on separate subnet

### PR DESCRIPTION
**Pull request by pierreprinetti**
OpenShift 4.6 does not support adding a MachineSet on a separate subnet.
Removing the corresponding use case from the baremetal-workers
documentation.

Also adding a use case for installing a cluster on bare metal machines
exclusively.

### CODE REVIEW

File: docs/user/openstack/deploy_baremetal_workers.md [hunk 1]
```diff
@@ -83,8 +83,7 @@ Initial cluster is deployed using bare-metal workers only. Bare-metal workers ar
 a preexisting network.
 
 - Requirements:
-
-    - An OpenStack subnet has been pre-provisioned. The subnet supports attaching both VMs and bare-metal servers to it.
+    - An OpenStack subnet has been pre-provisioned whish supports attaching both VMs and bare-metal servers to it.
```
Comment (mandre) [inline docs/user/openstack/deploy_baremetal_workers.md:86]: which

File: docs/user/openstack/deploy_baremetal_workers.md [hunk 2]
```diff
@@ -127,26 +126,28 @@ a preexisting network.
...
-Initial cluster deployment is done with only
-VM workers attached to the installer-provisioned network. The bare-metal
-workers are added after initial installation. Bare-metal workers are
-attached to a preexisting network. Traffic between masters and workers is routed between subnets.
+Initial cluster is deployed using bare-metal machines only for both Control
+Plane and Compute nodes. The cluster is deployed to a preexisting network.
 
 - Requirements:
-    - Cloud provider is configured to route traffic between VM subnets and bare-metal subnets.
+    - An OpenStack subnet has been pre-provisioned whish supports attaching bare-metal servers to it.
```
Comment (mandre) [inline docs/user/openstack/deploy_baremetal_workers.md:135]: Again :)

File: docs/user/openstack/deploy_baremetal_workers.md (hunk 1 above)
Comment (pierreprinetti) [reply docs/user/openstack/deploy_baremetal_workers.md]: I don't underschtand whash you mean

**Review by mandre — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**